mirall.desktop.in

```diff
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Categories=Utility;X-SuSE-SyncUtility;
 Type=Application
-Exec=@APPLICATION_EXECUTABLE@
+Exec=@APPLICATION_EXECUTABLE@ --showsettings
```

Comment:
But why do we want to show the settings dialog when starting owncloud from the menu?

Reply:
Well it gives the user a feedback that something happened.
I'd also expect 99% of the users to have it in autostart.